src/webgl-surface/drawing/texture/atlas-manager.ts

```diff
@@ -313,7 +313,7 @@ export class AtlasManager {
 
       else {
         // Log an error
-        console.error(`Could not fit image into atlas ${image.imagePath}`);
+        console.error(`Could not fit image into atlas:`, image.image);
```

Comment:
Please don't use ticks if they're not needed. We should have a linter rule for this. I'm not sure why it didn't get caught.